README.md

```diff
@@ -305,11 +333,28 @@ workflow in action. Make sure your GitHub repository has `AWS_ACCESS_KEY_ID` and
 `AWS_SECRET_ACCESS_KEY` secrets that can access the same account that you
 synthesized against.
 
-## Not supported yet
+> Here, we do want to commit `cdk.out`. This is not a recommended use-case; it is because
```

Comment:
What is this trying to say? I would just steer away from recommending this.

Reply:
In the 'tutorial', the user is running the example app from a different repository, and the example app has these options set. I imagine many people start out by trying this and/or copying the example app entirely, which is bad, so I want to build awareness on it. I'm trying to say *don't do this* here, so I'll adjust it to be a bit more forceful.